go/test/endtoend/cluster/cluster_process.go

```diff
@@ -187,6 +188,7 @@ func (cluster *LocalProcessCluster) StartKeyspace(keyspace Keyspace, shardNames
 				cluster.Hostname,
 				cluster.TmpDirectory,
 				cluster.VtTabletExtraArgs)
+			tablet.Alias = tablet.vttabletProcess.TabletPath
```

Comment:
when does the TabletPath get set and what is it set to? Alias is usually of the form cell1-00000001 (cell + uid).

Reply:
Yes, you are right, tabletpath is set as `cell1-000000001` [here](https://github.com/planetscale/vitess/blob/tal_test_vtctld/go/test/endtoend/cluster/vttablet_process.go#L181).